test/Interop/Cxx/value-witness-table/witness-lifetime-operations-irgen.swift

```diff
@@ -0,0 +1,48 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir -I /Users/zoecarver/Developer/swift-source/uswift/build-windows/swift -target x86_64-unknown-windows-msvc | %FileCheck %s
```

Comment:
`-I` and `-target` probably shouldn't be here

Reply:
Thanks, Egor!